backend/routes/users.js

```diff
@@ -102,11 +168,12 @@ router.delete("/:userId", function(req, res) {
   );
 });
 
+// TODO: remove when frotend is updated
 // Create a new user in firebase and mongodb
```

Comment:
I will assume update user will be used for adding things like the preferred name. Since we will no longer enter it with create-user.

Reply:
Yep, I'm thinking update user or a new patch method for the name?

Reply:
I think update user is fine...